docs/tutorial/keyboard-shortcuts.md

```diff
@@ -66,32 +66,60 @@ app.whenReady().then(() => {
 }).then(createWindow)
 ```
 
+> NOTE: In the code above, the `CommandOrControl` combination uses `Command`
+on macOS and `Control` on Windows/Linux.
+
 After launching the Electron application, if you press the defined key
-combination and then open the Console, you will see that Electron loves global
-shortcuts!
+combination then open the terminal that you ran your Electron application from,
+you will see that Electron loves global shortcuts!
 
 ### Shortcuts within a BrowserWindow
 
-#### Overview
+#### Using web APIs
 
-If you want to handle keyboard shortcuts for a [BrowserWindow], you can use the
-`keyup` and `keydown` event listeners on the window object inside the renderer
-process.
+If you want to handle keyboard shortcuts within a [BrowserWindow], you can
+listen for the `keyup` and `keydown` [DOM events][dom-events] inside the
+renderer process using the [addEventListener() API][addEventListener-api].
 
 ```js
 window.addEventListener('keyup', doSomething, true)
 ```
 
-Note the third parameter `true` which means the listener will always receive
+Note the third parameter `true` indicates that the listener will always receive
 key presses before other listeners so they can't have `stopPropagation()`
 called on them.
 
+#### Intercepting events in the main process
+
 The [`before-input-event`](../api/web-contents.md#event-before-input-event) event
 is emitted before dispatching `keydown` and `keyup` events in the page. It can
 be used to catch and handle custom shortcuts that are not visible in the menu.
 
-If you don't want to do manual shortcut parsing there are libraries that do
-advanced key detection such as [mousetrap].
+##### Example
+
+Starting with a working application from the
+[Quick Start Guide](quick-start.md), update the `main.js` file with the
+following lines:
+ 
+```js
+const { app, BrowserWindow } = require('electron')
+
+app.whenReady().then(() => {
+  let win = new BrowserWindow({ width: 800, height: 600 })
+  
+  win.webContents.on('before-input-event', (event, input) => {
+    // For example, only enable application menu keyboard shortcuts when
+    // Ctrl/Cmd are down.
+    win.webContents.setIgnoreMenuShortcuts(!input.control && !input.meta)
```

Comment:
This example from the API docs shows how to ignore menu shortcuts here, but not what to do with the `before-input-event` handler.

A base example that shows a new user what this does is maybe to `console.log` here like in the other examples, and use `event.preventDefault` like proposed in the API docs.